ide/textmate.lexer/external/binaries-list

```diff
@@ -15,4 +15,4 @@
 # specific language governing permissions and limitations
 # under the License.
 23D2F2EFF7FA0CDA465D86EC9D8BAB53E496D9E6 org.jruby.joni:joni:2.2.1
```

Comment:
Please consider bumping joni to 2.2.6. There were some bugfixes in there and to my understanding regexps are the heart of textmate grammars and thus performance and correctness for that area matter.

Reply:
Done.